influxdb/resultset.py

```diff
@@ -91,14 +91,14 @@ def __len__(self):
     def keys(self):
         keys = []
         for serie in self._get_series():
-            keys.append((serie['name'], serie['tags']))
+            keys.append((serie.get('name', None), serie.get('tags', None)))
```

Comment:
In which cases would `name` not be present? Same question for `tags`? Is it spec'd that no tags returns an empty list from the InfluxDB API: in which case that missing key should raise an exception instead of returning confusing data `(None, None)`, wouldn't you think?